client/state/invites/selectors.js

```diff
@@ -113,3 +113,29 @@ export function isRequestingResend( state, siteId, inviteId ) {
 export function didResendSucceed( state, siteId, inviteId ) {
 	return 'success' === get( state, [ 'invites', 'requestingResend', siteId, inviteId ], false );
 }
+
+/**
+ * Returns true if currently deleting an invite for the given site and
+ * invite ID, or false otherwise.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {String}  inviteId Invite ID
+ * @return {Boolean}          Whether invites resend is being requested
+ */
+export function isDeleting( state, siteId, inviteId ) {
+	return 'requesting' === get( state, [ 'invites', 'deleting', siteId, inviteId ], false );
+}
+
+/**
+ * Returns true if currently deleting an invite for the given site and
+ * invite ID, or false otherwise.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {String}  inviteId Invite ID
+ * @return {Boolean}          Whether invites resend is being requested
+ */
+export function didDeletionSucceed( state, siteId, inviteId ) {
+	return 'success' === get( state, [ 'invites', 'deleting', siteId, inviteId ], false );
+}
```

Comment:
I would rename these to `isDeletingInvite` and `didInviteDeletionSucceed`.

Arguably, the same should also be done for `isRequestingResend` and `didResendSucceed` above, so I would also be fine with leaving this alone, or doing all of those renames in a separate PR.

Reply:
Yeah, I think I just followed the existing names, but your suggestions are a little clearer - I'll make the change.